Makefile

```diff
@@ -62,6 +62,10 @@ mini-init: setup pipenv-dev-install react-init protobuf
 # Build frontend into static files.
 frontend: react-build
 
+.PHONY: frontendFast
+# Build frontend into static files without TypeChecking and EsLint.
+frontendFast: react-build-fast
```

Comment:
Nit: let's use snake case (`frontend-fast` instead of `frontendFast`) to match the rest of the Makefile

Reply:
Thank you, it's updated 👍